api/tests.py

```diff
@@ -1,3 +1,158 @@
 from django.test import TestCase
+from django.urls import reverse
+from django.contrib.gis.geos import Point
+from django.contrib.auth.models import User
+
+import json
+
+from infrastructure.models import ProjectStatus
+from infrastructure.factories import ProjectFactory
+from locations.models import GeometryStore, PointGeometry
+
+
+class TestProjectStatusListView(TestCase):
+    def test_that_all_project_statuses_are_returned(self):
+        url = reverse('api:project-status-list')
+        response = self.client.get(url)
+        self.assertEqual(len(json.loads(response.content.decode())), len(ProjectStatus.STATUSES))
+
+
+class TestGeometryStoreDetailView(TestCase):
+
+    def setUp(self):
+        self.geometry_store = GeometryStore()
+        self.geometry_store.save()
+        point = PointGeometry(geom=Point(30, 40))
+        point.save()
+        self.geometry_store.points.add(point)
+        self.geometry_store.save()
+        self.url = reverse('api:geometrystore-detail', args=[self.geometry_store.identifier])
+        self.project = ProjectFactory()
```

Comment:
It isn't clear from the tests why this setup is needed. What happens if there isn't a project?